### DISCUSSION TITLE
LinkPeaks function not functioning in AWS RStudio server

### DISCUSSION THREAD

**Discussion by chsnys** [stuart-lab/signac#1697] [General]
Hi , I am using signac 1.13.0 in my AWS Rstudio. I am also using the same version for my RStudio Desktop application. For the same code in RStudio desktop version, Linkpeaks function is  initially giving output on what that function is working on and finally giving output whereas in AWS Rstudio server, it is not giving any info on what the function is working and also not giving output even after hours of waiting( i even initialized verbose=TRUE). The RAM i have for my AWS Server is 128Gb.

My Mac(Desktop) details:

mac os:14.4.1 (23E224)
Chip     : Apple M1 Max

**Comment by timoast** [comment]
issue opened here: https://github.com/stuart-lab/signac/issues/1706
